xapian-applications/omega/templates/query

```diff
@@ -106,7 +106,7 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
+<td><b><a href="$html{omega.cgi?FMT=$url{clicklog}&QUERYID=$url{$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}&DOCID=$url{$id}&URL=$url{$field{url}}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
```

Comment:
It will be almost identical yes, which is why I'm suggesting only changing that bit to be a conditional. Have you looked at how we switch between the anyall variants?